README.rst

```diff
@@ -125,7 +123,7 @@
 -  CMake (version 3.15 or later)
 -  a C compiler (for system feature detection and the test helper binary)
 -  PCRE2 (headers and libraries) - optional, this will be downloaded if missing
--  gettext (headers and libraries) - optional, for translation support
```

Comment:
In future, we should add something like a "status locale" command,
that shows which localizations fish uses, and why.
I wonder what else we can do for discoverability.